server/util/authutil/authutil.go

```diff
@@ -26,3 +30,30 @@ func AuthorizeGroupRole(u interfaces.UserInfo, groupID string, allowedRoles role
 	}
 	return nil
 }
+
+// EffectiveGroup returns the group membership effective for the request
+// context, given an authenticated user.
+func EffectiveGroup(ctx context.Context, u interfaces.UserInfo) (*interfaces.GroupMembership, error) {
```

Comment:
Could we call this SelectedGroup? Or were you avoiding that because for group-based-auth nothing is selected?

Reply:
Yeah, I figured for group-based auth it may be confusing to call it SelectedGroup.